run_markdownlint.sh

```
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Temporary script to run markdownlint on docs directory
+echo "Running markdownlint-cli2 on docs/*.md"
+echo "========================================="
+
+# Change to project directory
+cd "/Users/daelegbe/Documents/Projects/claude-config"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)